tests/__mocks__/constants.ts

```diff
@@ -0,0 +1,114 @@
+export const TIME_LABELS = [
+  {
+    name: "Time: <1 Hour",
+  },
+  {
+    name: "Time: <2 Hours",
+  },
+  {
+    name: "Time: <4 Hours",
+  },
+  {
+    name: "Time: <1 Day",
+  },
+  {
+    name: "Time: <1 Week",
+  },
+];
+
+export const PRIORITY_LABELS = [
+  {
+    name: "Priority: 1 (Normal)",
+  },
+  {
+    name: "Priority: 2 (Medium)",
+  },
+  {
+    name: "Priority: 3 (High)",
+  },
+  {
+    name: "Priority: 4 (Urgent)",
+  },
+  {
+    name: "Priority: 5 (Emergency)",
+  },
+];
+
+export const PRICE_LABELS = [
+  { name: "Price: 12.5 USD", color: "1f883d" },
+  { name: "Price: 25 USD", color: "1f883d" },
+  { name: "Price: 37.5 USD", color: "1f883d" },
+  { name: "Price: 50 USD", color: "1f883d" },
+  { name: "Price: 62.5 USD", color: "1f883d" },
+  { name: "Price: 75 USD", color: "1f883d" },
+  { name: "Price: 125 USD", color: "1f883d" },
+  { name: "Price: 100 USD", color: "1f883d" },
+  { name: "Price: 150 USD", color: "1f883d" },
+  { name: "Price: 200 USD", color: "1f883d" },
+  { name: "Price: 250 USD", color: "1f883d" },
+  { name: "Price: 300 USD", color: "1f883d" },
+  { name: "Price: 400 USD", color: "1f883d" },
+  { name: "Price: 450 USD", color: "1f883d" },
+  { name: "Price: 500 USD", color: "1f883d" },
+  { name: "Price: 600 USD", color: "1f883d" },
+  { name: "Price: 750 USD", color: "1f883d" },
+  { name: "Price: 800 USD", color: "1f883d" },
+  { name: "Price: 1000 USD", color: "1f883d" },
+];
+
+export const CONFIG_PATH = ".github/ubiquibot-config.yml";
+export const UBIQUITY = "ubiquity";
+export const USER_2 = "user2";
+export const TEST_REPO = "test-repo";
+export const SHA_1 = "1234";
+export const CONFIG_CHANGED_IN_COMMIT = ".github/ubiquibot-config.yml was modified or added in the commits";
+export const CREATED_NEW_LABEL = "Created new price label";
+export const UPDATING_ISSUE_1_IN_TEST_REPO = "Updating issue 1 in test-repo";
+export const UPDATING_ISSUE_2_IN_TEST_REPO = "Updating issue 2 in test-repo";
+export const UPDATING_ISSUE_3_IN_TEST_REPO = "Updating issue 3 in test-repo";
+export const UPDATING_FROM_1_TO_5 = "Updating base rate from 1 to 5";
+export const CREATING_MISSING_LABELS = "Creating missing labels done";
+export const PUSHER_NOT_AUTHED = "Pusher is not an admin or billing manager";
+export const SENDER_NOT_AUTHED = "Sender is not an admin or billing manager";
+export const NO_RECOGNIZED_LABELS = "No recognized labels to calculate price";
+export const NEEDS_TRIGGERED_BY_ADMIN_OR_BILLING_MANAGER = "Changes should be pushed and triggered by an admin or billing manager.";
```

Comment:
Perhaps these can be organized in an object called `strings` and placed in a separate file.

Reply:
Might be overkill but tools like `i18n` could be userful for this.

Reply:
I think that might be more appropriate for user facing code. As I understand this is commonly used for translations.

Reply:
I do not know if these strings are exposed to the user. I just thought about it because it is easy to put them all in a file, and also because they support templates (so adding arguments like number) and singular / plural etc. Seems to much for this case but `"Updating issue 1 in test-repo"` could be a template for example to avoid copy pasting 3 times.

Reply:
> I do not know if these strings are exposed to the user

I guess all logs are exposed to the user so that could be an idea across all core plugins maybe?

Reply:
It's one of the few plugins that are very verbose with `conversation-rewards` so not needed for every plugin.

Reply:
perhaps handle those that need it as one separate task then

Reply:
This being said, these strings are only used for testing right? So you could just have them in a `json` as @0x4007 suggested, just to avoid piling them in the code (you can directly import the JSON in the code which should be easy to use).

Reply:
> This being said, these strings are only used for testing right

Well yeah the specifics like "test-repo" are for tests only but the production logs would be the same with the actual payload info. So ideally if adding translations etc we'd have a collection of string literals with injectables vars and translate those

Resolved already with an object: `STRINGS` vs json